README.md

```diff
@@ -3,23 +3,24 @@
 # THIS IS ALPHA SOFTWARE
 
 Sirepo is a scientific application framework, currently for particle accelator codes.
-The codes run inside a Python web server.
+Sirepo runs inside a Python web server.
 
-Learn more at sireop.com.
+Learn more at sireop.com(coming soon).
 
 Documentation: http://rssynergia.readthedocs.org/en/latest/
 
 ### Vagrant Install
 
-You can start Sirepo with Vagrant. First create a Vagrantfile:
+You can start Sirepo with [Vagrant](https://www.vagrantup.com/).
+
+First create a Vagrantfile:
 
 ```bash
-cat > Vagrantfile <<'EOF'
+//Vagrantfile
```

Comment:
Actually, the syntax doesn't work!
Just tested it and it throws a syntax error. It was intended to be a comment.

Vagrant files are actually parsed as ruby, so it should be:

# Vagrantfile

It's totally minor, but in how-to step-by-step documentation, I include the file name at the top of the snippet as a comment. That way you can easily write sequential snippets that jump between files following the logical flow of the thinking and instructions. It's usefully concise. Totally minor in this case, but it's a reflex now.

It's something I picked up while drafting some blog content in objective-C where every class has a .h and .m file, and coherent step-by-step iterative development in one file usually requires changes in the other.

Nick

> On Mar 22, 2016, at 8:26 AM, Rob Nagler <notifications@github.com> wrote:
> 
> In README.md:
> 
> >  
> >  ```bash
> > -cat > Vagrantfile <<'EOF'
> > +//Vagrantfile
> Thanks, Nick! I'm not familiar with //Vagrantfile syntax. How does it work?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>